gql/client.py

```diff
@@ -27,6 +28,8 @@
     validate,
 )
 
+from gql.transport.data_structures.graphql_request import GraphQLRequest
```

Comment:
I would prefer to have a flat folder structure. Please move the `graphql_request.py` file in the `gql` folder and remove the `data_structures` folder. Also `GraphQLRequest` could be imported directly from gql.

Reply:
whoops, my mistake you're completely right!